{"name": "Успешная аутентификация курьера при вводе валидных данных", "status": "failed", "statusDetails": {"message": "AssertionError: assert (404 == 200)\n +  where 404 = <Response [404]>.status_code", "trace": "self = <test_courier_login.TestCourierLogin object at 0x0000027605918B90>\n\n    @allure.title('Успешная аутентификация курьера при вводе валидных данных')\n    @allure.description('Код и тело ответа.')\n    def test_courier_login_success(self):\n        response = requests.post(Urls.URL_courier_login, data=Data.valid_courier_data)\n>       assert response.status_code == 200 and 'id' in response.text\nE       assert (404 == 200)\nE        +  where 404 = <Response [404]>.status_code\n\ntests\\test_courier_login.py:15: AssertionError"}, "description": "Код и тело ответа.", "start": 1742320156541, "stop": 1742320156625, "uuid": "ed4d5ba5-5100-44a3-b2cc-c3d437cfc2c5", "historyId": "c29840593483f1bfb3d305fecaa73dbe", "testCaseId": "c29840593483f1bfb3d305fecaa73dbe", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-JHHR6FB"}, {"name": "thread", "value": "1556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}